lint.js

```diff
@@ -88,6 +88,7 @@ const main = (str, callback) => {
     .catch(function(err) {
         options.status = 'rejected';
         console.warn(err);
+        process.exitCode = 0;
```

Comment:
Unsure why this would consume other `errorCode`s

Reply:
Switched to `process.exit(0)` but its the same thing in this instance.